app/src/main/res/values/settings_keys.xml

```diff
@@ -851,4 +851,32 @@
         <item>ZM</item>
         <item>ZW</item>
     </string-array>
+
+    <!-- Limit mobile data usage  -->
+    <string name="limit_mobile_data_usage_key" translatable="false">limit_mobile_data_usage</string>
+    <string name="limit_mobile_data_usage_value" translatable="false">@string/limit_data_usage_none_key</string>
+    <string-array name="limit_data_usage_description_list">
+        <item>@string/limit_data_usage_none_description</item>
+        <item>1080p60</item>
+        <item>1080p</item>
+        <item>720p60</item>
+        <item>720p</item>
+        <item>480p</item>
+        <item>360p</item>
+        <item>240p</item>
+        <item>144p</item>
+    </string-array>
+    <string-array name="limit_data_usage_values_list">
```

Comment:
Almost, resolution_list_values is mostly identical except for the first item: "Best resolution" vs "No limit". I was also bothered by the duplication but I didn't know of a way to share *some* of the items without more refactoring and I was trying to avoid too many code changes for my first push. We may also want to consider a "No video" item at some point.

Reply:
hm ok.